<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatForm</class>
 <widget class="QWidget" name="ChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Chat Room Application</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTextEdit" name="nickname_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="client_list">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>232</width>
     <height>149</height>
    </rect>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chat_body">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>731</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="log_in_button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>21</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log in to chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Change nickname</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="message_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="say_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Say</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Now in chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leave_chat_button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>310</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leave chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="chat_name_lable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Chat name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
